rust/lance-index/src/vector.rs

```diff
@@ -55,6 +55,10 @@ pub struct Query {
     /// The number of probes to load and search.
     pub nprobes: usize,
 
+    /// The number of candidates to reserve while searching.
+    /// this is an optional parameter for HNSW related index types.
+    pub ef: Option<usize>,
```

Comment:
Why `ef`?  I assume it is from the literature but is it short for something?

Reply:
It stands for "explore factor", 'ef' may be more common